web/controllers/coping_strategy_controller.ex

```diff
@@ -38,7 +38,8 @@ defmodule Healthlocker.CopingStrategyController do
     changeset = Ecto.Changeset.put_change(changeset, :user_id, user_id)
 
     case Repo.insert(changeset) do
-      {:ok, _post} ->
+      {:ok, post} ->
+        conn |> track_created(post)
```

Comment:
We might want to change up `track_created` so we don't need to pass in the conn. It makes it a bit confusing having:
```
conn |> track_created(post)
```
and
```
conn
|> put_flash...
```

The functions under the latter `conn` do something to the conn then return it, so it makes sense having that piped through. `track_created` doesn't interact with the conn at all, and just needs the user, so perhaps we can pass that in instead.